Documentation updates for latest AI changes. (#371)
spiceai/docs: spiceaidocs/docs/features/configurable-llms/index.md

---
title: 'Configuring Language Models'
sidebar_label: 'Configuring LLMs'
description: 'Learn how to configure language models in Spice.'
sidebar_position: 7
pagination_prev: null
pagination_next: null
---

> Spice handles using language models (LLMs) from a variety of sources (see [model components](/components/models/index.md)) and provides additional customisation on how inference will be performed in the Spice runtime. This includes:

Reviewer: ```suggestion
Spice supports language models (LLMs) from several sources (see [model components](/components/models/index.md)) and provides configuration for how inference will be performed in the Spice runtime. This includes:
```
